include/itkReplaceFeatureMapNanInfImageFilter.hxx

```diff
@@ -20,6 +20,7 @@
 
 #include "itkReplaceFeatureMapNanInfImageFilter.h"
 
+
 namespace itk
```

Comment:
Since we are in the `itk` namespace, we do not need to use `itk::` below (and it is not used per ITK style guidelines).